### PR TITLE
Update elpi

### PR DESCRIPTION
**Pull request by gares**
###### Motivation for this change

Update elpi and coq-elpi

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by gares** [comment]
@GrahamcOfBorg build coqPackages_8_11.coq-elpi coqPackages_8_11.hierarchy-builder coqPackages_8_12.coq-elpi coqPackages_8_12.hierarchy-builder coqPackages_8_13.coq-elpi coqPackages_8_13.hierarchy-builder

**Comment by gares** [comment]
@GrahamcOfBorg build coqPackages_8_11.coq-elpi coqPackages_8_11.hierarchy-builder coqPackages_8_12.coq-elpi coqPackages_8_12.hierarchy-builder coqPackages_8_13.coq-elpi coqPackages_8_13.hierarchy-builder

**Comment by gares** [comment]
@vbgl This should be ready now

File: pkgs/development/ocaml-modules/elpi/default.nix [hunk 1]
```diff
@@ -5,6 +5,7 @@
 with lib;
 let fetched = import ../../../build-support/coq/meta-fetch/default.nix
   {inherit lib stdenv fetchzip; } ({
+    release."1.13.1".sha256 = "12a9nbdvg9gybpw63lx3nw5wnxfznpraprb0wj3l68v1w43xq044";
```
Comment (vbgl) [inline pkgs/development/ocaml-modules/elpi/default.nix:8]: Why isn’t it the default version (the one mentioned on line 3 of this file)?

Comment (gares) [reply pkgs/development/ocaml-modules/elpi/default.nix:8]: I did not notice there was a default version

**Pull request merged**